difftool-dune.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# difftool-dune.sh
+
+echo "Arguments received: $@"
+file="$2"
+if [ -z "$file" ]; then
+    file="$1"
+fi
+echo "File to diff: $file"
+
+# Check if file exists in the specified location or in the current directory
+if [ ! -f "$file" ]; then
+    if [ -f "$(basename "$file")" ]; then
```

Comment:
Can you give an example that require this ?
Also, diffcessible can do its own error checks, no need to add more.